chromium_src/chrome/installer/mini_installer/mini_installer_constants.cc

```diff
@@ -69,14 +69,18 @@ const wchar_t kClientStateKeyBase[] =
 const wchar_t kCleanupRegistryKey[] =
     L"Software\\Microsoft\\Windows\\CurrentVersion\\Uninstall\\Brave";
 #else
+
+// The product name in the strings below must be the same as kProductPathName.
+
 // The path to the key containing each app's Clients registry key.
 // No trailing slash on this one because the app's GUID is not appended.
-const wchar_t kClientsKeyBase[] = L"Software\\Chromium";
+const wchar_t kClientsKeyBase[] = L"Software\\Brave-Browser-Development";
 // The path to the key containing each app's Client State registry key.
 // No trailing slash on this one because the app's GUID is not appended.
-const wchar_t kClientStateKeyBase[] = L"Software\\Chromium";
+const wchar_t kClientStateKeyBase[] = L"Software\\Brave-Browser-Development";
 // The path to the key in which kCleanupRegistryValue is found.
-const wchar_t kCleanupRegistryKey[] = L"Software\\Chromium";
+const wchar_t kCleanupRegistryKey[] = L"Software\\Brave-Browser-Development";
```

Comment:
are these changes intentional? should they stay after delta updates PR will be reverted eventually?
maybe land this separately?